addons/test_discuss_full/tests/test_performance.py

```diff
@@ -24,7 +24,7 @@ class TestDiscussFullPerformance(HttpCase):
     _query_count = 48 + 1  # +1 is necessary to fix nondeterministic issue on runbot
     # Queries for _query_count_discuss_channels:
     #     1: bus last id
-    _query_count_discuss_channels = 69
+    _query_count_discuss_channels = 78
```

Comment:
not a good sign to have that kind of increase for a code move, there is probably something not correctly done

Reply:
`setupUp` makes 9 discuss channels, counter increases by 9, so likely a linear increase by channels?